apps/web/components/apps/InstallAppButtonChild.tsx

```diff
@@ -112,7 +112,7 @@ export const InstallAppButtonChild = ({
                   );
                 }}>
                 <p>
-                  {team.name} {isInstalled && `(${t("installed")})`}
+                  {t(team.name)} {isInstalled && `(${t("installed")})`}
```

Comment:
The `team.name` will cover "me" so it'll grab the translation. Any other team name will be passed through as normal. @Ud